environment.yml

```diff
@@ -0,0 +1,260 @@
+  - parso=0.8.3=pyhd8ed1ab_0
+  - pcre2=10.40=hc3806b6_0
+  - pexpect=4.8.0=pyh1a96a4e_2
+  - pickleshare=0.7.5=py_1003
+  - pillow=9.2.0=py39hf3a2cdf_3
+  - pip=22.3.1=pyhd8ed1ab_0
+  - pixman=0.40.0=h36c2ea0_0
+  - pkgutil-resolve-name=1.3.10=pyhd8ed1ab_0
+  - platformdirs=2.5.2=pyhd8ed1ab_1
+  - plotly=5.11.0=pyhd8ed1ab_0
+  - ply=3.11=py_1
+  - prompt-toolkit=3.0.33=pyha770c72_0
+  - psutil=5.9.4=py39hb9d737c_0
+  - pthread-stubs=0.4=h36c2ea0_1001
+  - ptyprocess=0.7.0=pyhd3deb0d_0
+  - pulseaudio=16.1=h126f2b6_0
+  - pure_eval=0.2.2=pyhd8ed1ab_0
+  - py-opencv=4.6.0=py39hcca971b_6
+  - pycparser=2.21=pyhd8ed1ab_0
+  - pygments=2.13.0=pyhd8ed1ab_0
+  - pyopenssl=22.1.0=pyhd8ed1ab_0
+  - pyparsing=3.0.9=pyhd8ed1ab_0
+  - pyqt=5.15.7=py39h18e9c17_2
+  - pyqt5-sip=12.11.0=py39h5a03fae_2
+  - pyrsistent=0.19.2=py39hb9d737c_0
+  - pysocks=1.7.1=pyha2e5f31_6
+  - python=3.9.15=hba424b6_0_cpython
+  - python-dateutil=2.8.2=pyhd8ed1ab_0
+  - python-fastjsonschema=2.16.2=pyhd8ed1ab_0
+  - python_abi=3.9=3_cp39
+  - pytorch=1.12.1=cpu_py39h3439074_1
+  - pyzmq=24.0.1=py39headdf64_1
+  - qt-main=5.15.6=he99da89_3
+  - readline=8.1.2=h0f457ee_0
+  - requests=2.28.1=pyhd8ed1ab_1
+  - scipy=1.9.3=py39hddc5342_2
+  - setuptools=65.5.1=pyhd8ed1ab_0
+  - sip=6.7.5=py39h5a03fae_0
+  - six=1.16.0=pyh6c4a22f_0
+  - sleef=3.5.1=h9b69904_2
+  - soupsieve=2.3.2.post1=pyhd8ed1ab_0
+  - stack_data=0.6.2=pyhd8ed1ab_0
+  - svt-av1=1.3.0=h27087fc_0
+  - tbb=2021.7.0=h924138e_0
+  - tenacity=8.1.0=pyhd8ed1ab_0
+  - tk=8.6.12=h27826a3_0
+  - toml=0.10.2=pyhd8ed1ab_0
+  - torchvision=0.13.0=cpu_py39hedfcc36_0
+  - tornado=6.2=py39hb9d737c_1
+  - tqdm=4.64.1=pyhd8ed1ab_0
+  - traitlets=5.6.0=pyhd8ed1ab_0
+  - typing_extensions=4.4.0=pyha770c72_0
+  - tzdata=2022g=h191b570_0
+  - unicodedata2=15.0.0=py39hb9d737c_0
+  - urllib3=1.26.13=pyhd8ed1ab_0
+  - wcwidth=0.2.5=pyh9f0ad1d_2
+  - wheel=0.38.4=pyhd8ed1ab_0
+  - widgetsnbextension=4.0.3=pyhd8ed1ab_0
+  - x264=1!164.3095=h166bdaf_2
+  - x265=3.5=h924138e_3
+  - xcb-util=0.4.0=h166bdaf_0
+  - xcb-util-image=0.4.0=h166bdaf_0
+  - xcb-util-keysyms=0.4.0=h166bdaf_0
+  - xcb-util-renderutil=0.3.9=h166bdaf_0
+  - xcb-util-wm=0.4.1=h166bdaf_0
+  - xorg-fixesproto=5.0=h7f98852_1002
+  - xorg-inputproto=2.3.2=h7f98852_1002
+  - xorg-kbproto=1.0.7=h7f98852_1002
+  - xorg-libice=1.0.10=h7f98852_0
+  - xorg-libsm=1.2.3=hd9c2040_1000
+  - xorg-libx11=1.7.2=h7f98852_0
+  - xorg-libxau=1.0.9=h7f98852_0
+  - xorg-libxdmcp=1.1.3=h7f98852_0
+  - xorg-libxext=1.3.4=h7f98852_1
+  - xorg-libxfixes=5.0.3=h7f98852_1004
+  - xorg-libxi=1.7.10=h7f98852_0
+  - xorg-libxrender=0.9.10=h7f98852_1003
+  - xorg-renderproto=0.11.1=h7f98852_1002
+  - xorg-xextproto=7.3.0=h7f98852_1002
+  - xorg-xproto=7.0.31=h7f98852_1007
+  - xz=5.2.6=h166bdaf_0
+  - zeromq=4.3.4=h9c3ff4c_1
+  - zipp=3.11.0=pyhd8ed1ab_0
+  - zlib=1.2.13=h166bdaf_4
+  - zstd=1.5.2=h6239696_4
+  - pip:
+    - pycolmap==0.3.0
+prefix: /home/nadir/anaconda3/envs/StudProject
```

Comment:
Do not commit local setting files (add to ignore). This file contains your local path and this path doesn't have any sense for another developer. Modify it to exclude path or exclude it from repo.